static/app/views/organizationContext.spec.tsx

```diff
@@ -105,53 +87,41 @@ describe('OrganizationContext', function () {
     expect(getTeamsMock).toHaveBeenCalled();
   });
 
-  it('does not fetch if organization is already set', async function () {
```

Comment:
`it('does not fetch if organization is already set'` this test is being removed. Not sure how we'd support that with react query.  I also can't figure out when that would happen.